composer.json

```diff
@@ -119,20 +119,17 @@
         "symfony/var-dumper": "~3.4.0",
         "symfony/browser-kit": "~3.4.0",
         "symfony/dom-crawler": "~3.4.0",
-
-        "liip/functional-test-bundle": "~3.3.0",
         "mautic/transifex": "^4.0",
         "http-interop/http-factory-guzzle": "^1.0",
         "webfactory/exceptions-bundle": "~4.3",
         "friendsofphp/php-cs-fixer": "~2.16.1",
         "liip/test-fixtures-bundle": "^1.6",
         "phpstan/phpstan": "0.12.25",
         "rector/rector-prefixed": "0.6.13",
-        "theofidry/psysh-bundle": "^v3.5.1",
         "symfony/phpunit-bridge": "~5.1.0",
-        "phpunit/phpunit": "~8.5.0",
-        "phpunit/phpunit-selenium": "~8.0.0",
```

Comment:
AFAIK we're not using Selenium but are looking into Cypress instead now, so I removed the dev dependency.